packages/playground/src/util/codeHandlers.ts

```diff
@@ -0,0 +1,69 @@
+import svelte from 'svelte/compiler';
+
+export const addSnippetToCode = (
+  code: string,
+  snippet: string,
+  end: number | null,
```

Comment:
Is this a line number? If so, let's be explicit with the naming. Consider using char number instead of line number.

Please, use the AST also here. To be sure that we place the snippet at proper place (not in the middle of a tag).